os/board/rtl8730e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -595,6 +595,7 @@ int8_t cmd_wifi_disconnect(void)
 void cmd_wifi_info(int argc, char **argv)
 {
 	int i = 0;
+	u8 *ifname[2] = {(unsigned char*)WLAN0_NAME, (unsigned char*)WLAN1_NAME};
 	u8 wlan_idx[NET_IF_NUM];// Declared the array size as follow the NET_IF_NUM
```

Comment:
I am not sure if this is the right change. It will be better if someone from network team or realtek makes this change because just a few lines below lines they have mentioned,
```
// Need to change to use WLAN_IDX instead of ifname if using this API
```
also loop goes from ```0``` to ```NET_IF_NUM - 1```, but you have hard coded the array size as 2. Even if it works now, in future if ```NET_IF_NUM``` changes, this will cause problems.

Reply:
Let us revert this ifname change and change the line 608 as below:

`nvdbg("\n\r\nWIFI %s Status: Running", ifname[i]);`

to

`nvdbg("\n\r\nWIFI idx %d Status: Running", i);`